kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -72,6 +74,8 @@ def __init__(self, app, port=5000, brain=None, allowed_cors_origin=False):
         self.app.add_url_rule('/synapses/start/order', view_func=self.run_synapse_by_order, methods=['POST'])
         self.app.add_url_rule('/synapses/start/audio', view_func=self.run_synapse_by_audio, methods=['POST'])
         self.app.add_url_rule('/shutdown/', view_func=self.shutdown_server, methods=['POST'])
+        self.app.add_url_rule('/mute/', view_func=self.get_mute, methods=['GET'])
+        self.app.add_url_rule('/mute/', view_func=self.set_mute, methods=['POST'])
```

Comment:
unmute missing
self.app.add_url_rule('/unmute/', view_func=self.set_mute, methods=['POST'])
or ```unmute``` useless ?

Reply:
Yes it's useless